flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -3229,5 +3229,9 @@
     },
     "net.rpcs3.RPCS3": {
         "module-rpcs3-source-git-branch": "Needed for custom updater to update once a day per upstream recommendation."
+    },
+    "io.github.loot.loot": {
+	"finish-args-unnecessary-xdg-config-access": "Needed to detect games installed through Heroic Games Launcher.",
+        "finish-args-unnecessary-xdg-data-access": "Needed to detect and make changes to games installed through Steam."
```

Comment:
Please fix the indentation here

![image](https://github.com/flathub-infra/flatpak-builder-lint/assets/62639087/3fc7256a-9b5b-40bc-8224-96531e672fba)

Reply:
Done, I didn't notice GitHub's editor was using tabs with width 8...